test/image_test.py

```diff
@@ -255,6 +256,28 @@ def test_save(self):
                 # clean up the temp file, comment out to leave tmp file after run.
                 os.remove(temp_filename)
 
+    def test_save_to_fileobject(self):
+        s = pygame.Surface((1, 1))
+        s.fill((23, 23, 23))
+        bytes_stream = io.BytesIO()
+
+        pygame.image.save(s, bytes_stream)
+        bytes_stream.seek(0)
+        s2 = pygame.image.load(bytes_stream, "tga")
+        self.assertEqual(s.get_at((0, 0)), s2.get_at((0, 0)))
+
+    def test_save_tga(self):
+        s = pygame.Surface((1, 1))
+        s.fill((23, 23, 23))
+        try:
+            temp_filename = "%s.tga" % "tmpimg"
```

Comment:
Perhaps use the tempfile.NamedTemporaryFile here?
This way the file will be written in a writable area.

```python
with tempfile.NamedTemporaryFile(delete=False) as afile:
    temp_filename = afile.name
```

Reply:
Excellent suggestion! I was not sure tempfile was a thing in Python 2.7. But when adding it, I noticed it was already use elsewhere. :)

I had just copied over what the previous test was doing as a starting point.

I've made the changes and squashed my commit.